### ISSUE TITLE
Medical Treatement- PAK revives people if they get downed after pak-ing starts

### ISSUE THREAD

**Issue by ofetios** [acemod/ACE3#9793]
**Mods (complete and add to the following information):**
- **Arma 3:** `2.14`
- **CBA:** `3.16.1.231025`
- **ACE3:** `3.16.3` 

**Description:**
If you start pak-ing an unit and in the 10 seconds he goes unconscious but you're still pak-ing him he will wake up.

**Steps to reproduce:**
_Follow [https://ace3.acemod.org/img/wiki/user/issue_flowchart.webp](this flowchart)!_

1. Hurt a unit
2. Heal the unit
3. Start using a PAK on the unit and have someone else shoot that unit until he goes unconscious 

**Expected behavior:**
If he goes unconscious the pak timer should stop and the person should not wake up

**Where did the issue occur?**
- Dedicated

**Log Files:**
- I do not have log files for this

**Additional context:**
In order to PAK the code only checks if you can use a PAK right now, therefor unless the person dies you can fully heal him. 

**Screenshots:**
Non applicable

**Comment by BaerMitUmlaut** [comment]
The PAK is applied at the end of the interaction, just like any other interaction, for example wheel changes.

**Comment by ofetios** [comment]
Yes but currently the game checks if you can pak someone and starts the interaction, but doesn't check if you are actually supposed to be able to pak someone at the end. This allows you to fully revive someone who is downed as long as you start pak-ing him before he goes unconscious. This is indeed a rare occurrence but I thought I should report this as it probably affects a lot of other things as well.
